core/domain/topic_domain.py

```diff
@@ -221,14 +236,107 @@ def validate(self):
                     % (skill_id, self.id))
 
 
+class SubtopicPage(object):
+    """Domain object for a Subtopic page."""
+
+    def __init__(self, subtopic_page_id, topic_id, html_data, language_code):
+        """Constructs a SubtopicPage domain object.
+
+        Args:
+            subtopic_page_id: str. The unique ID of the subtopic page.
+            topic_id: str. The topic that this subtopic is a part of.
+            html_data: str. The html data that describes this subtopic.
```

Comment:
The HTML content of the subtopic page.

(It's not just "describing". It's explaining a set of skills.)

Reply:
Done.